src/utils/codeowners.ts

```diff
@@ -97,24 +97,21 @@ const isValidCodeownersGlob = (glob: string) => {
   // as well as the gitignore pattern format which it extends
   // https://git-scm.com/docs/gitignore#_pattern_format
 
-  if (glob.startsWith('!')) {
-    // A pattern cannot use ! to negate
-    return false;
-  }
-  if (glob.includes('{') || glob.includes('}')) {
-    // A pattern cannot use { } for brace expansion or brace sets
-    return false;
-  }
-  if (glob.startsWith('\\#')) {
-    // A pattern cannot start with an escaped # using \ so it is treated as a pattern and not a comment
-    return false;
-  }
-  if (isGlob(glob) && CHARACTER_RANGE_PATTERN.test(glob)) {
-    // A pattern cannot use [ ] to define a character range
-    return false;
-  }
-
-  return true;
+  const isGlobString = isString(glob);
+  const isNotEmpty = glob.length > 0;
```

Comment:
The if statement for these are left on line 90. That statement should probably be removed.

Reply:
oh my bad! copy pasta confusion. doing it.

Reply:
done!